src/ophys_etl/modules/dff/__main__.py

```diff
@@ -46,22 +56,31 @@ def compute_dff_trace(corrected_fluorescence_trace: np.ndarray,
         `corrected_fluorescence_trace`.
     """
     sigma_f = noise_std(corrected_fluorescence_trace, short_filter_length)
-
+    inactive_trace = corrected_fluorescence_trace.copy()
     # Long timescale median filter for baseline subtraction
-    baseline = medfilt(corrected_fluorescence_trace, long_filter_length)
+    if inactive_percentile > 0 or inactive_percentile < 100:
+        low_baseline = percentile_filter(
+            corrected_fluorescence_trace, size=long_filter_length,
+            percentile=inactive_percentile,
+            )
+        active_mask = corrected_fluorescence_trace > (
+            low_baseline + 3 * sigma_f)
+        inactive_trace[active_mask] = float('nan')
```

Comment:
Why not just `np.nan`?

Reply:
I thought they're equivalent so I defaulted to the one that native to python. I don't really have a preference either way.